v2/internal/pkg/clusterresources/clusterresources.go

```diff
@@ -832,3 +845,11 @@ func (o *ClusterResourcesGenerator) GenerateSignatureConfigMap(allRelatedImages
 
 	return nil
 }
+
+func generateOcMirrorAnnotations() map[string]string {
+	return map[string]string{
+		"createdBy":         "oc-mirror v2",
+		"createdAt":         time.Now().UTC().Format(time.RFC850),
+		"oc-mirror_version": version.Get().GitVersion,
```

Comment:
The version issue should be fixed by https://github.com/openshift/oc-mirror/pull/1109